iothub/device/samples/PnpDeviceSamples/PnpHelper/PnpHelper.cs

```diff
@@ -150,6 +150,16 @@ public static string CreatePnpCommandName(string commandName, string componentNa
             return string.IsNullOrWhiteSpace(componentName) ? commandName : $"{componentName}*{commandName}";
         }
 
+        /// <summary>
+        /// Create the DPS payload to provision a device as plug and play.
+        /// </summary>
+        /// <param name="modelId">The model Id corresponding to the device.</param>
+        /// <returns>The DPS payload to provision a device as plug and play.</returns>
+        public static string CreateDpsPayload(string modelId)
+        {
+            return $"{{ \"modelId\": \"{modelId}\" }}";
```

Comment:
parameter checking? Not just null or whitespace, but it should follow a certain format too.

Reply:
I'll add a check for null or whitespace. The format of DTDL model Id is being validated by the service.